File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/PeticionCodificacionDeNuevoTermino.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for PeticionCodificacionDeNuevoTermino complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="PeticionCodificacionDeNuevoTermino">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element name="establecimiento" type="{http://www.w3.org/2001/XMLSchema}string"/>
 *         &lt;element name="idConcepto" type="{http://www.w3.org/2001/XMLSchema}string"/>
 *         &lt;element name="termino" type="{http://www.w3.org/2001/XMLSchema}string"/>
 *         &lt;element name="tipoDescripcion" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="esSensibleAMayusculas" type="{http://www.w3.org/2001/XMLSchema}boolean" minOccurs="0"/>
 *         &lt;element name="email" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="observacion" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="profesional" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="profesion" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="especialidad" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="sub-especialidad" type="{http://www.w3.org/2001/XMLSchema}string"/>
 *         &lt;element name="categoria" type="{http://www.w3.org/2001/XMLSchema}string"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "PeticionCodificacionDeNuevoTermino", propOrder = {
    "establecimiento",
    "idConcepto",
    "termino",
    "tipoDescripcion",
    "esSensibleAMayusculas",
    "email",
    "observacion",
    "profesional",
    "profesion",
    "especialidad",
    "subEspecialidad",
    "categoria"
})
public class PeticionCodificacionDeNuevoTermino {

    @XmlElement(required = true)
    protected String establecimiento;
    @XmlElement(required = true)
    protected String idConcepto;
    @XmlElement(required = true)
    protected String termino;
    @XmlElement(defaultValue = "Preferida")
    protected String tipoDescripcion;
    @XmlElement(defaultValue = "false")
    protected Boolean esSensibleAMayusculas;
    protected String email;
    protected String observacion;
    protected String profesional;
    protected String profesion;
    protected String especialidad;
    @XmlElement(name = "sub-especialidad", required = true)
    protected String subEspecialidad;

    @XmlElement(required = true)
    protected String categoria;

    /**
     * Gets the value of the establecimiento property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getEstablecimiento() {
        return establecimiento;
    }

    /**
     * Sets the value of the establecimiento property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setEstablecimiento(String value) {
        this.establecimiento = value;
    }

    /**
     * Gets the value of the idConcepto property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getIdConcepto() {
        return idConcepto;
    }

    /**
     * Sets the value of the idConcepto property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setIdConcepto(String value) {
        this.idConcepto = value;
    }

    /**
     * Gets the value of the termino property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getTermino() {
        return termino;
    }

    /**
     * Sets the value of the termino property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setTermino(String value) {
        this.termino = value;
    }

    /**
     * Gets the value of the tipoDescripcion property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getTipoDescripcion() {
        return tipoDescripcion;
    }

    /**
     * Sets the value of the tipoDescripcion property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setTipoDescripcion(String value) {
        this.tipoDescripcion = value;
    }

    /**
     * Gets the value of the esSensibleAMayusculas property.
     * 
     * @return
     *     possible object is
     *     {@link Boolean }
     *     
     */
    public Boolean isEsSensibleAMayusculas() {
        return esSensibleAMayusculas;
    }

    /**
     * Sets the value of the esSensibleAMayusculas property.
     * 
     * @param value
     *     allowed object is
     *     {@link Boolean }
     *     
     */
    public void setEsSensibleAMayusculas(Boolean value) {
        this.esSensibleAMayusculas = value;
    }

    /**
     * Gets the value of the email property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getEmail() {
        return email;
    }

    /**
     * Sets the value of the email property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setEmail(String value) {
        this.email = value;
    }

    /**
     * Gets the value of the observacion property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getObservacion() {
        return observacion;
    }

    /**
     * Sets the value of the observacion property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setObservacion(String value) {
        this.observacion = value;
    }

    /**
     * Gets the value of the profesional property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getProfesional() {
        return profesional;
    }

    /**
     * Sets the value of the profesional property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setProfesional(String value) {
        this.profesional = value;
    }

    /**
     * Gets the value of the profesion property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getProfesion() {
        return profesion;
    }

    /**
     * Sets the value of the profesion property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setProfesion(String value) {
        this.profesion = value;
    }

    /**
     * Gets the value of the especialidad property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getEspecialidad() {
        return especialidad;
    }

    /**
     * Sets the value of the especialidad property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setEspecialidad(String value) {
        this.especialidad = value;
    }

    /**
     * Gets the value of the subEspecialidad property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getSubEspecialidad() {
        return subEspecialidad;
    }

    /**
     * Sets the value of the subEspecialidad property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setSubEspecialidad(String value) {
        this.subEspecialidad = value;
    }

    /**
     * Gets the value of the categoria property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getCategoria() {
        return categoria;
    }

    /**
     * Sets the value of the categoria property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setCategoria(String value) {
        this.categoria = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/PeticionCodificacionDeNuevoTermino.php
<?php

namespace Semantikos\ClientBundle\API;


class PeticionCodificacionDeNuevoTermino
{

    /**
     * @var string $establecimiento
     * @Type("string")
     */
    public $establecimiento = null;

    /**
     * @var string $idConcepto
     * @Type("string")
     */
    public $idConcepto = null;

    /**
     * @var string $termino
     * @Type("string")
     */
    public $termino = null;

    /**
     * @var string $tipoDescripcion
     * @Type("string")
     */
    public $tipoDescripcion = null;

    /**
     * @var boolean $esSensibleAMayusculas
     * @Type("string")
     */
    public $esSensibleAMayusculas = null;

    /**
     * @var string $email
     * @Type("string")
     */
    public $email = null;

    /**
     * @var string $observacion
     * @Type("string")
     */
    public $observacion = null;

    /**
     * @var string $profesional
     * @Type("string")
     */
    public $profesional = null;

    /**
     * @var string $profesion
     * @Type("string")
     */
    public $profesion = null;

    /**
     * @var string $especialidad
     * @Type("string")
     */
    public $especialidad = null;

    /**
     * @var string $subespecialidad
     * @SerializedName("sub-especialidad")
     * @Type("string")
     */
    public $subespecialidad = null;

    /**
     * @var string $categoria
     * @Type("string")
     */
    public $categoria = null;

    /**
     * @param string $establecimiento
     * @param string $idConcepto
     * @param string $termino
     * @param string $subespecialidad
     * @param string $categoria
     */
    public function __construct()
    {
    }

    /**
     * @return string
     */
    public function getEstablecimiento()
    {
      return $this->establecimiento;
    }

    /**
     * @param string $establecimiento
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setEstablecimiento($establecimiento)
    {
      $this->establecimiento = $establecimiento;
      return $this;
    }

    /**
     * @return string
     */
    public function getIdConcepto()
    {
      return $this->idConcepto;
    }

    /**
     * @param string $idConcepto
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setIdConcepto($idConcepto)
    {
      $this->idConcepto = $idConcepto;
      return $this;
    }

    /**
     * @return string
     */
    public function getTermino()
    {
      return $this->termino;
    }

    /**
     * @param string $termino
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setTermino($termino)
    {
      $this->termino = $termino;
      return $this;
    }

    /**
     * @return string
     */
    public function getTipoDescripcion()
    {
      return $this->tipoDescripcion;
    }

    /**
     * @param string $tipoDescripcion
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setTipoDescripcion($tipoDescripcion)
    {
      $this->tipoDescripcion = $tipoDescripcion;
      return $this;
    }

    /**
     * @return boolean
     */
    public function getEsSensibleAMayusculas()
    {
      return $this->esSensibleAMayusculas;
    }

    /**
     * @param boolean $esSensibleAMayusculas
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setEsSensibleAMayusculas($esSensibleAMayusculas)
    {
      $this->esSensibleAMayusculas = $esSensibleAMayusculas;
      return $this;
    }

    /**
     * @return string
     */
    public function getEmail()
    {
      return $this->email;
    }

    /**
     * @param string $email
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setEmail($email)
    {
      $this->email = $email;
      return $this;
    }

    /**
     * @return string
     */
    public function getObservacion()
    {
      return $this->observacion;
    }

    /**
     * @param string $observacion
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setObservacion($observacion)
    {
      $this->observacion = $observacion;
      return $this;
    }

    /**
     * @return string
     */
    public function getProfesional()
    {
      return $this->profesional;
    }

    /**
     * @param string $profesional
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setProfesional($profesional)
    {
      $this->profesional = $profesional;
      return $this;
    }

    /**
     * @return string
     */
    public function getProfesion()
    {
      return $this->profesion;
    }

    /**
     * @param string $profesion
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setProfesion($profesion)
    {
      $this->profesion = $profesion;
      return $this;
    }

    /**
     * @return string
     */
    public function getEspecialidad()
    {
      return $this->especialidad;
    }

    /**
     * @param string $especialidad
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setEspecialidad($especialidad)
    {
      $this->especialidad = $especialidad;
      return $this;
    }

    /**
     * @return string
     */
    public function getSubespecialidad()
    {
      return $this->subespecialidad;
    }

    /**
     * @param string $subespecialidad
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setSubespecialidad($subespecialidad)
    {
      $this->subespecialidad = $subespecialidad;
      return $this;
    }

    /**
     * @return string
     */
    public function getCategoria()
    {
      return $this->categoria;
    }

    /**
     * @param string $categoria
     * @return PeticionCodificacionDeNuevoTermino
     */
    public function setCategoria($categoria)
    {
      $this->categoria = $categoria;
      return $this;
    }

}
